README.md

```diff
@@ -1,3 +1,7 @@
+
+WARNING: This step is deprecated, use the AVD Manager step instead:
```

Comment:
It is rather a feedback but now it looks like:
![screen shot 2019-02-25 at 10 35 12 am](https://user-images.githubusercontent.com/1564933/53327897-1c02ed00-38e9-11e9-8447-96207d2287d0.png)
And in one of our already deprecated steps it looks like: 
![screen shot 2019-02-25 at 10 35 05 am](https://user-images.githubusercontent.com/1564933/53327915-2624eb80-38e9-11e9-979c-1cb121249630.png)

Reply:
The removal date is missing.

Reply:
Fixed in:
https://github.com/bitrise-steplib/steps-create-android-emulator/pull/27/commits/bc65b77f432cc2c9b5a009ce7f78c59bbf51bc32